week-7/Homework/mandatory/1-debugging-practice/script.js

```diff
@@ -37,8 +37,8 @@ function submit() {
     alert("Please fill all fields!");
     return false;
   } else {
-    let book = new Book(title.value, title.value, pages.value, check.checked);
-    library.push(book);
+    let book = new Book(title.value, author.value, pages.value, check.checked);
+    myLibrary.push(book);
     render();
   }
 }
```

Comment:
Altom there is a bug in the input field, it is submitting even though the author field is empty. work on that and fix it.